### ISSUE TITLE
block comments in python not working?

### ISSUE THREAD

**Issue by egradman** [NaturalDocs/NaturalDocs#21]
Hi, just getting started with NaturalDocs but running into a problem the interwebs says I shouldn't be having.

My python block comments aren't being processed.  I've altered the python language definition as follows:

```
Alter Language: Python
  Block Comment: """ """
```

It worked when I did this (and adjusted my comments to match):

```
Alter Language: Python
  Block Comment: """/* */"""
```

Does there need to be asymmetry between the start and end tokens of a block comment?

**Comment by NaturalDocs** [comment]
Maybe.  I didn't design for the possibility of them being the same, nor was I aware of that capability in Python, so I don't know how the code would react to that.  I suspect the fix will be fairly easy so if that ends up being the case I'll post a new version here in the next few days.

**Comment by NaturalDocs** [comment]
Here, try this version: http://www.naturaldocs.org/temp/Natural_Docs_temp6.zip

The changes will be included in the next full release of Natural Docs.

**Comment by NaturalDocs** [comment]
These changes were included in 2.0.2.
